php/Terminus/Dispatcher/Subcommand.php

```diff
@@ -166,10 +167,10 @@ private function validate_args( $args, $assoc_args, $extra_args ) {
 
     $cmd_path = implode( ' ', get_path( $this ) );
     foreach ( $validator->get_unknown() as $token ) {
-      \Terminus::warning( sprintf(
-        "The `%s` command has an invalid synopsis part: %s",
-        $cmd_path, $token
-      ) );
+      \Terminus::get_logger()->warning(
```

Comment:
This style of syntax will blow up some earlier-but-still-supported version of PHP. The logger needs to be assigned to a variable first.